app/services/task-service.ts

```diff
@@ -89,6 +95,25 @@ export class TaskService extends ServiceBase {
         return getOnceProxy(this.get(jobId, taskId, options));
     }
 
+    public getMultiple(jobId: string, taskIds: string[], properties?: string): Observable<Task[]> {
```

Comment:
I thought you said to return an observable with the task[] data in it rather than using the ListProxy